lib/content/audio/predictions.ex

```diff
@@ -88,7 +90,9 @@ defmodule Content.Audio.Predictions do
           minutes: predictions.minutes,
           verb: if(src.terminal?, do: :departs, else: :arrives),
           track_number: Content.Utilities.stop_track_number(predictions.stop_id),
-          platform: src.platform
+          platform: src.platform,
+          station_code: predictions.station_code,
+          zone: predictions.zone
```

Comment:
Note for onlookers, since this wasn't obvious to me at first: Previously the check in the audio code path wasn't even getting hit on <20 min predictions, because we weren't passing this data, but that didn't matter because the check was only relevant for 20+ min anyway. Now that it's lower, we need to feed this data through so the check gets hit all the time.